packages/monaco/src/browser/style/index.css

```diff
@@ -1,9 +1,9 @@
 /* FIXME when https://github.com/Microsoft/monaco-editor/issues/113#issuecomment-240406949 is fixed */
```

Comment:
@kpge please remove all comments then

Have you checked that you cannot reproduce https://github.com/microsoft/monaco-editor/issues/113#issue-171635379 anymore?